odoo/addons/base/models/ir_model.py

```diff
@@ -346,6 +352,27 @@ class IrModelFields(models.Model):
                                                       "    name, partner_id.name")
     store = fields.Boolean(string='Stored', default=True, help="Whether the value is stored in the database.")
 
+    @api.depends('relation', 'relation_field')
+    def _compute_relation_m2o(self):
+        for rec in self:
+            if rec.state == 'manual':
```

Comment:
add the condition `and rec.relation_field`
We should not search for anything if there is nothing to search for :wink: